model/fn-execution/src/main/proto/beam_fn_api.proto

```diff
@@ -516,7 +516,7 @@ message Elements {
   repeated Data data = 1;
 
   // (Optional)  A list of timer byte streams.
-  repeated Timer timer = 2;
+  repeated Timer timers = 2;
```

Comment:
@robertwb did you want to rename this field or the proto message Timer -> Timers or both?

Reply:
Both.